### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by techwritermat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,247 @@
-# swift-quickstart-platform
+# PubNub Swift Sample App
 
 This repository contains the files required to run the [PubNub Swift Quickstart](https://www.pubnub.com/docs/platform/quickstarts/swift).
```
Comment (hadesara) [inline README.md:3]: Do we still need this line?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,247 @@
...
+PubNub takes care of the infrastructure and APIs needed for the realtime communication layer of your application. Work on your app's logic and let PubNub handle sending and receiving data across the world in less than 100ms.
+
+## Get keys
+
+You will need the publish and subscribe keys to authenticate your app. Get your keys from the [Admin Portal](https://dashboard.pubnub.com/login).
+
+## Setup the project
+
+If you don't want to copy the code from this document, you can clone the repository and use the files in there.
+
```
Comment (hadesara) [inline README.md:14]: 'and use the files in there', can we say something better here? like 'clone the repository and start the xcode project' or something similar.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,247 @@
...
+2. Give it a Product Name, `pnquickstart`.
+3. Set the Language to Swift.
+4. Set User Interface to SwiftUI.
+5. Set Lifecycle to UIKitAppDelegate.
+6. Save the project to your file system.
+7. Navigate to File > Swift Packages > Add Package Dependency, enter the package repository URL: https://github.com/pubnub/swift.git and click the dialog's Next and Finish buttons, keeping all default options.
+
+## Add project files
+
+If you don't want to copy the code from this document, you can clone the repository and use the files in there.
```
Comment (hadesara) [inline README.md:26]: Duplicate line

**Review by hadesara — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (polarweasel) [reply README.md:3]: I say yes. You're almost certainly arriving here from the Quickstart page on the platform docs. This is a quiet reminder that you're arrived in the right spot. Probably also doesn't hurt for SEO purposes, either.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,247 @@
...
+
+PubNub takes care of the infrastructure and APIs needed for the realtime communication layer of your application. Work on your app's logic and let PubNub handle sending and receiving data across the world in less than 100ms.
+
+## Get keys
+
+You will need the publish and subscribe keys to authenticate your app. Get your keys from the [Admin Portal](https://dashboard.pubnub.com/login).
+
+## Setup the project
+
+If you don't want to copy the code from this document, you can clone the repository and use the files in there.
```
Comment (polarweasel) [inline README.md]:
```suggestion
If you don't want to copy the code from this document, you can clone this repository and use the project files locally.
```

File: README.md (hunk 2 above)
Comment (polarweasel) [reply README.md:14]: Suggestion above.

File: README.md (hunk 3 above)
Comment (polarweasel) [reply README.md:26]: We're duplicating this one on purpose, in all the READMEs. Good reminder given we're about to ask you to copy-paste almost 200 lines. :)

**Review by crimsonred — APPROVED** [review]

**Pull request merged**